src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -1674,6 +1674,30 @@ public function testPartialDenormalizationWithInvalidVariadicParameter()
             DenormalizerInterface::COLLECT_DENORMALIZATION_ERRORS => true,
         ]);
     }
+
+    public function testExtractPublicPropertiesFirst()
```

Comment:
This test must be long to the `AbstractObjectNormalizerTest` instead.
Also, you might add a test in the `ReflectionExtractorTest` as well.